content/applications/sales/crm/acquire_leads/email_manual.rst

```diff
@@ -0,0 +1,121 @@
+
+The email received by the alias is added to the *chatter* thread for the lead. The subject line of
+the message is added to the title field, and the :guilabel:`Email` field is updated with the
+contact's email address.
+
+.. image:: email_manual/chatter-message.png
+   :align: center
+   :alt: The chatter thread of a newly created lead in the CRM app.
+
+.. note::
+   If the *leads* feature is **not** enabled on the database, messages to the email alias are added
+   to the database as opportunities.
+
+.. seealso::
+   :doc:`../../../general/email_communication/email_servers`
+
+Manually create leads
+=====================
+
+Leads can be added directly to the *CRM* app by manually creating a new record. Navigate to
+:menuselection:`CRM app --> Leads` to view a list of existing leads.
+
+.. tip::
+   Leads can also be added via the :doc:`Generate Leads <lead_mining>` button.
+
+At the top-left of the list, click :guilabel:`New` to open a blank :guilabel:`Leads` form.
+
+In the first field of the new form, enter a title for the new lead. Next, enter a :guilabel:`Contact
+Name`, and a :guilabel:`Company Name`.
+
+.. note::
+   If a lead is :doc:`converted to an opportunity <convert>`, the :guilabel:`Company Name` field is
+   used to either link this opportunity to an existing customer, or to create a new customer.
+
+Manually create opportunities
+-----------------------------
+
+To manually create an opportunity, navigate to :menuselection:`CRM app --> Sales --> My Pipeline`.
+At the top-left of the page, click :guilabel:`New` to create a new opportunity Kanban card. In the
+:guilabel:`Organization/Contact` field, enter the name of the company the opportunity is for.
+
+Choose a name, and enter it in the :guilabel:`Opportunity` field. *This is a required field.* When
+manually creating an opportunity, it is helpful to add a name that relates to the details of the
+opportunity.
+
+.. example::
+   In the example below, the opportunity is named `5 VP Chairs`. This identifies the product the
+   customer is interested in, as well as the potential number of products.
+
+   .. image:: email_manual/opportunity-example.png
+      :align: center
+      :alt: An example of an opportunity in the CRM pipeline.
+
+Enter the contact information for the opportunity in the :guilabel:`Email` and :guilabel:`Phone`
+fields.
+
+In the :guilabel:`Expected Revenue` field, enter an estimated value for the opportunity. Then, use
+the |st-o| :guilabel:`(star)` icons to assign a priority.
+
+- |st-o| |st-o| |st-o|: low priority
+- |st| |st-o| |st-o|: medium priority
+- |st| |st| |st-o|: high priority
+- |st| |st| |st|: very high priority
+
+.. note::
+   The information in the :guilabel:`Expected Revenue` and priority fields can be used to track
+   performance for individual salespeople, and on a team basis. See
+   :doc:`../performance/expected_revenue_report` and :doc:`../track_leads/lead_scoring` for more
+   information.
```

Comment:
Suggestion to add a note about how the priority effects the order of leads in the Kanban view :)
```suggestion
.. note::
   Assigning a priority changes the order of leads in Kanban view, with higher priority leads
   displayed first.
```